### ISSUE TITLE
Misleading log in MemoryDB mode

### ISSUE THREAD

**Issue by gumb0** [ethereum/aleth#5814]
```
> aleth/aleth --db memorydb -v 4
aleth, a C++ Ethereum client
INFO  11-11 17:30:37 aleth net    Id: ##77047807…
INFO  11-11 17:30:37 aleth net    ENR: [ seq=6 id=v4 key=02770478… ip=178.19.221.38 tcp=30303 udp=30303 ]
TRACE 11-11 17:30:37 aleth trace  Opened blockchain DB. Latest: #d4e56740…(rebuild not needed)
TRACE 11-11 17:30:37 aleth statedb Opening state database: "/home/andrei/.ethereum/d4e56740/12041/state"
```

Does StateDB still use disk in `--db memorydb` mode? I think it doesn't, and then the log message should be fixed, this path doesn't tell anything about reality.

**Comment by halfalicious** [comment]
Agreed, will fix

**Comment by halfalicious** [comment]
Fixed by #5829
